odoo/addons/base/models/ir_actions_report.py

```diff
@@ -410,6 +413,55 @@ def _prepare_html(self, html, report_model=False):
 
         return bodies, res_ids, header, footer, specific_paperformat_args
 
+    @api.model
+    def _build_wkhtmltoimage_args(self, width, height, allow_js=False):
```

Comment:
Zboing @xmo-odoo  (could consider delegating, but I know you love frameworky - security stuff). In short: we need to use wkhtmltoimage in order to generate images (yay) based on records (i.e. we generate a card people can share on social networks, like speakers for OXP). Feel free to check with @reth-odoo and me for more complete details (won't summarize everything here :D ) .

We are in early review stage (don't consider reviewing the whole diff completely) but already having a look at framework stuff is probably a good idea if we want to make it for the freeze (somehow).

Reply:
about that :eyes:

Reply:
Friendly reminder @xmo-odoo , you have probably missed it, because I know you're a big fan of everything wkhtml-related 😄